bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -214,9 +217,11 @@ public BatchExecutor(
    */
   ListenableFuture<com.google.bigtable.v1.Row> issueGetRequest(Get get) {
     LOG.trace("issueGetRequest(Get)");
-    ReadRowsRequest.Builder builder = getAdapter.adapt(get);
```

Comment:
My thinking is that it is / will be error prone (e.g., when making this change it'd have been easy to miss a call-site that should have been fixed)